django/contrib/postgres/fields/citext.py

```diff
@@ -1,8 +1,20 @@
-from django.db.models import CharField
+from django.db.models import CharField, EmailField, TextField
 
-__all__ = ['CITextField']
+__all__ = ['CIText', 'CICharField', 'CIEmailField', 'CITextField']
 
 
-class CITextField(CharField):
+class CIText:
```

Comment:
What about making it a `CITextField(models.Field)` so it's usuable as is? It would still be reusable for subclassing.